docs/overview/roadmap.md

```diff
@@ -1 +1,58 @@
 # Roadmap
+
+This document describes roadmap of **ONE** project.
+
+This project **ONE** aims at providing a high-performance, on-device neural network (NN) inference
+framework that performs inference of a given NN model on processors, such as CPU, GPU, DSP, or NPU, 
+in the target platform, such as Tizen, Android, and Ubuntu. 
+
+## Progress
+
+Until last year, we already saw significant gains in accelerating with a single CPU or GPU backend.
+We have seen better performance improvements, not only when using a single backend, but even when
+mixing CPUs or GPUs considering the characteristics of individual operations. It could give us an
+opportunity to have a high degree of freedom in terms of operator coverage, and possibly provide
+better performance compared to single backend acceleration.
+
+On the other hand, we introduced the compiler as a front-end. This will support a variety of deep
+learning frameworks in relatively spacious host PC environments, while the runtime running on the
+target device is intended to take a smaller burden. In this approach, the compiler and the runtime
+will effectively share information among themselves by the Common IR, named _circle_, and a
```

Comment:
_circle_ is all in lower cases. I used _Circle_  with capital letter _C_ to denote it's not just a circle (round object) but a proper noun. What do you think of using _Circle_ ?

Reply:
@seanshpark , from a long time ago, I generally used _itallic_ for all of our proper nouns. _NN Package_, _nnfw_, _nncc_, etc. for example. Only **ONE** uses the bold boldface, recently. Also, I prefer to express everything in lowercase letters whenever possible. We do not have explicit guidance on this. This is one of the important documents to be added.

If you aren't against it, what if all the documents I write are unified according to the rules above? If the guidelines are confirmed afterwards, it is good to reflect them in a batch.

Reply:
OK, sounds good :)